README.md

```diff
@@ -1 +1,174 @@
-# Budget-App
\ No newline at end of file
+<a name="readme-top"></a>
+
+<div align="center">
+  <h3><b>BUDGET APP</b></h3>
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+- [Diagram](#diagram)   
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# Blog App <a name="about-project"></a>
+
+
+**Tracker Budget** is a mobile web application where you can manage your budget: There is a list of transactions associated with a category, so that you can see how much money you spent and on what.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li><a href="https://www.ruby-lang.org/en/">Ruby</a></li>
+  </ul>
+</details>
+
+<p align="right"><a href="#readme-top">back to top</a></p>
+
+### ERD Diagram <a name="diagram">
+![Preview](./app/assets/images/erd_diagram.png)
+
+
+
+
+### Live Demo
+This is a [live link](https://rails-budget-app-mnta.onrender.com/) demo of the mobile app
+
+
+### Video Demo
+This is a [video presentation](https://www.loom.com/share/a38fbc8f9ca242529c1ac3e3a63e4285?sid=e66c70b8-858c-4b11-99bf-b24980cb078c) demonstration of the app
+
```

Comment:
- Hey, great adding the video, however, you recorded the video without any sound, You want to tell people about the project by explaining some features of the project and not visually showing people what you've done without saying anything which has defeated the purpose of the video. Please, record another video to ensure one can hear you. :pray:

Reply:
- [ ] You have done a great job addressing most of the changes @SabaAhmad404. But, This comment from the previous code reviewer is not been addressed yet, As it has been mentioned, the video content has no sound, This prevents the user from understanding the content and taking the point from it. Please, record another video to ensure one can hear you. 🙏
> * Hey, great adding the video, however, you recorded the video without any sound, You want to tell people about the project by explaining some features of the project and not visually showing people what you've done without saying anything which has defeated the purpose of the video. Please, record another video to ensure one can hear you. 🙏